libs/utils/Cargo.toml

```diff
@@ -19,7 +19,7 @@ thiserror = "1.0"
 tokio = { version = "1.17", features = ["macros"]}
 tokio-rustls = "0.23"
 tracing = "0.1"
-tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+tracing-subscriber = { version = "0.3", features = ["env-filter", "json", "registry"] }
```

Comment:
Is the registry needed after all?

Reply:
Nope, fixed. Actually didn't have much success trying to use that.